vocab/credentials/v2/vocabulary.yml

```diff
@@ -29,6 +29,12 @@ class:
     label: Credential status
     comment: A Credential Status provides enough information to determine the current status of the credential (for example, suspended or revoked). This class serves as a supertype for specific status types.
 
+  - id: ConfidenceMethod
```

Comment:
This appears to be reserving a "class" not a "predicate"... I guess we really need to add `Credential` or we will have a weird mismatch between how to handle classes in the spec.

Reply:
I do not understand what you are saying, @OR13. The vocabulary includes a number of "Classes" that serve as some sort of an "abstract" superclass. I fail to see how this has any relation to the presence or not of a separate class for credentials.

Reply:
I would prefer not to discuss this issue here. It is a valid question, but it would turn this PR into an endless discussion thread, going way beyond a simple addition to the vocabulary. Let us avoid that; this is what may lead to problems like https://github.com/w3c/vc-data-model/pull/1226#issuecomment-1665495765